src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -2389,17 +2389,17 @@ private static bool DatabaseNeedsToRunIdleOperations(DocumentDatabase database,
 
             if ((now - maxLastWork).TotalMinutes > 5)
```

Comment:
maxLastWork here is the last write on any env, right?

Reply:
yes

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-16252